gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -703,6 +729,10 @@ private File getOutputFile() {
         String dir = directory.getText();
         return dir.isEmpty() ? new File(name) : new File(dir, name);
       }
+
+      private static boolean isPerfetto(TraceTypeCapabilities config) {
```

Comment:
This assumes that a trace is either Perfetto, or gfxtrace by default. To be future proof of more than two types of traces, maybe we can introduce an enumeration of possible types of traces?

Reply:
Yes, this enum exists and this function is there to check that enum. You are of course correct that if we add another trace type, everywhere where `isPerfetto` is called would need to be changed. However, it would need to be changed anyways, so for now I'd rather just stick with this shorthand